mlflow/R/mlflow/tests/testthat/test-tracking-runs.R

```diff
@@ -133,9 +133,11 @@ test_that("mlflow_log_batch() works", {
   mlflow_clear_test_dir("mlruns")
   mlflow_start_run()
   mlflow_log_batch(
-    metrics = list(mse = 21, rmse = 42),
-    params = list(l1 = 0.01, optimizer = "adam"),
-    tags = list(model_type = "regression", data_year = "2015")
+    metrics = data.frame(key = c("mse", "mse", "rmse"), value = c(21, 23, 42),
+                         timestamp = c(100, 200, 300)),
```

Comment:
1. I think we should add a test case with step.
2. I think we should test what happens if there is a missing value in the dataframe, e.g.:
```
timestamp = c(100, NA, 300)
```